### DISCUSSION TITLE
Comparison with pyenchant

### DISCUSSION THREAD

**Discussion by Julian** [facelessuser/pyspelling#198] [Q&A]
Hi there. I'm finding this library mentioned in a post somewhere after being a long time user of [pyenchant](https://pypi.org/project/pyenchant/).

Do you have any opinions on comparisons between the two? Or yeah what made you write a new abstraction over the spell checkers rather than leveraging enchant, if anything.

(And thanks for writing this!)

**Comment by facelessuser** [comment]
1. I was unaware of enchant or pyenchant at the time.
2. I specifically wanted to be able to have a layer where  I could filter content based on custom plugins.  For instance, we have a plugin that allows me to filter HTML blocks out based on CSS selectors, or whatever context I desire to implement. This was the main driving force, that I could create whatever interface I needed to spell check the way I wanted to.

> Do you have any opinions on comparisons between the two?

I have no opinions per item 1) above. I say use what works best for you.

On a side note, is it possible we could leverage pyenchant in the future? Maybe, I'd have to evaluate if it would work in the flow. I was unaware of it before, but if it improved things for the user, it is always a possibility. Ultimately, I would require that retaining item 2) above would be of the utmost importance if I considered leveraging anything new in the future.

**Reply by Julian** [reply]
Thanks for the info! Appreciated! And thanks again for writing this!
